lib/generators/ops_care/templates/rake_tasks/opscare_data_mangling.rake

```diff
@@ -10,33 +10,31 @@
 
 namespace :opscare do
   namespace :data do
-
     def get_db_config(env)
-      db_config = Rails.application.config.database_configuration[env]
-      db_config
+      Rails.application.config.database_configuration[env]
     end
 
     desc "Mangle data for a safer usage in envs other than prod"
-    task mangle: :environment do |t,args|
+    task mangle: :environment do
       # Fail safe. Don't remove this test!
-      if ENV['RAILS_ENV'] == 'production'
+      if ENV["RAILS_ENV"] == "production"
         puts "You probably don't want to mangle production data..."
         puts "Exiting"
         exit
       end
 
       # Get the DB config if you're calling it directly
-      db = get_db_config(ENV['RAILS_ENV'])
+      # db = get_db_config(ENV["RAILS_ENV"])
```

Comment:
This line is not supposed to be commented out.
I imagine Rubocop complains about an unused variable.
This is a template/placeholder file. Commenting this out is putting form before function.

Reply:
Regardless of Rubocop, this is a template and this line is unused in its current form. So just like you would find in Rails `config/environments/*.rb` it should be commented out with a description of usage.